Extensions/PlatformBehavior/tests/platformerobjectruntimebehavior.spec.js

```diff
@@ -157,10 +157,11 @@ describe('gdjs.PlatformerObjectRuntimeBehavior', function () {
       object.getBehavior('auto1').simulateReleaseKey();
       for (let i = 0; i < 10; ++i) {
         runtimeScene.renderAndStep(1000 / 60);
+        expect(object.getBehavior('auto1').isFalling()).to.be(true);
       }
 
       //Check that the object is falling
-      expect(object.getY()).to.be(1.25);
+      expect(object.getY()).to.be(3.75);
```

Comment:
Any reason for this change? 🤔

Reply:
Yes, blame it, I put an explanation in the commit message, so you can see the change in the code too.

Reply:
Ok I see, the grabbing look behind looks ok, and the release key making the object fall at the same frame too, same for the ladder 👍